resources/js/store/actions.js

```diff
@@ -364,7 +364,17 @@ export const actions = {
     axios
     .post("/media-api/attach", value)
     .then(response => {
-      location.reload()
+      const myEvent = new CustomEvent("customEvent", {
+        detail: {
+          tag: value.tag,
+          data: response.data.data
+        },
+        bubbles: true,
+        cancelable: true,
+        composed: false,
+      });
+      document.getElementsByClassName("attach-media-listener")[0].dispatchEvent(myEvent);
```

Comment:
and then we can update that here as well: 
```suggestion
      document.getElementsByClassName("attach-media-listener")[0].dispatchEvent(attachEvent);
```